### PR TITLE
Update new OVA related information

### PR DESCRIPTION
**Pull request by pranavmalaviya**
A pass over the existing guides for Ubuntu 18.04 VM image since we have changed our approach from VMware as a builder to exporting OVA from the AWS EC2. This is also done to make the image available for public use.

### CODE REVIEW

File: pages/user guides/tidal-vms.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 keywords: vm, virtual, machine, image, ova, ovf, aws, ami, tidal, tools
-last_updated: November, 2021
+last_updated: January, 2022
 summary: "Virtual machines pre-configured with Tidal software"
 sidebar: main_sidebar
 permalink: pre-configured-vms.html
 ---
 
-To help you get your Tidal Migrations discovery phase up and running, we've created a series of pre-configured virtual machines with common operating systems. These VMs have the latest Tidal software and dependencies installed for you, so you can focus on getting things done.
+To help you get your Tidal Migrations discovery phase up and running, we've created a pre-configured virtual machines image. These VMs have the latest Tidal softwares and dependencies installed for you, so you can focus on getting things done.
```
Comment (przmv) [inline pages/user guides/tidal-vms.md]: “Software” is uncountable

**Review by przmv — CHANGES_REQUESTED** [review]
LGTM, just some minor fixes are required

File: pages/user guides/tidal-vms.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
 keywords: vm, virtual, machine, image, ova, ovf, aws, ami, tidal, tools
-last_updated: November, 2021
+last_updated: January, 2022
 summary: "Virtual machines pre-configured with Tidal software"
 sidebar: main_sidebar
 permalink: pre-configured-vms.html
 ---
 
-To help you get your Tidal Migrations discovery phase up and running, we've created a series of pre-configured virtual machines with common operating systems. These VMs have the latest Tidal software and dependencies installed for you, so you can focus on getting things done.
+To help you get your Tidal Migrations discovery phase up and running, we've created a pre-configured virtual machine images. These VMs have the latest Tidal softwares and dependencies installed for you, so you can focus on getting things done.
```
Comment (SamDesmondKing) [inline pages/user guides/tidal-vms.md:10]: Small grammar note, I think here you want `machine image` and `software` singular. Also `This VM has` etc if we're talking about a single one.

**Review by SamDesmondKing — APPROVED** [review]
Looks good Pranav!

File: pages/user guides/tidal-vms.md (hunk 2 above)
Comment (SamDesmondKing) [reply pages/user guides/tidal-vms.md:10]: Oh I just saw @przmv 's comment! Great minds think alike lol

**Review by jmariomejiap — APPROVED** [review]
Aside from Petr and Sam's comments, it looks good to go 🚀

File: pages/user guides/tidal-vms.md [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
 keywords: vm, virtual, machine, image, ova, ovf, aws, ami, tidal, tools
-last_updated: November, 2021
+last_updated: January, 2022
 summary: "Virtual machines pre-configured with Tidal software"
 sidebar: main_sidebar
 permalink: pre-configured-vms.html
 ---
 
-To help you get your Tidal Migrations discovery phase up and running, we've created a series of pre-configured virtual machines with common operating systems. These VMs have the latest Tidal software and dependencies installed for you, so you can focus on getting things done.
+To help you get your Tidal Migrations discovery phase up and running, we've created a pre-configured virtual machine image. This VM has the latest Tidal software and dependencies installed for you, so you can focus on getting things done.
```
Comment (przmv) [inline pages/user guides/tidal-vms.md:10]:
```suggestion
To help you get your Tidal Migrations discovery phase up and running, we've created a pre-configured virtual machine image. This VM has the latest Tidal Migrations software and dependencies installed for you, so you can focus on getting things done.
```

File: pages/user guides/tidal-vms.md [hunk 4]
```diff
@@ -1,30 +1,30 @@
 ---
 title: Pre-Configured Virtual Machines
 keywords: vm, virtual, machine, image, ova, ovf, aws, ami, tidal, tools
-last_updated: November, 2021
+last_updated: January, 2022
 summary: "Virtual machines pre-configured with Tidal software"
```
Comment (przmv) [inline pages/user guides/tidal-vms.md:5]:
```suggestion
summary: "Virtual machines pre-configured with Tidal Migrations software"
```

**Review by przmv — CHANGES_REQUESTED** [review]
“Tidal softvare” vs “Tidal Migrations software”

**Review by przmv — APPROVED** [review]
LGTM

**Pull request merged**